### ISSUE TITLE
Add tests to CI

### ISSUE THREAD

**Issue by dougiesquire** [ACCESS-NRI/yamanifest#15]
It doesn't look like the tests are run as part of the CI. Should we add them?

I'm not very familiar with TravisCI. Is it still a used/supported platform. Is it worth moving to Github Actions?

**Comment by aidanheerdegen** [comment]
Travis still exists, but I'd guess it is no longer "switched on" as it has changed it's funding and usage model.

Need to be ported to GitHub actions. I wonder if we shouldn't bite the bullet and do #13 as this is clearly a community asset that I probably don't have time to maintain.

**Comment by dougiesquire** [comment]
Good idea. Let me add a test to #14, we'll get that merged, and then I can move everything to the ACCESS-NRI org and switch to Github Actions.
